File: C/dicecup.c
#include <stdio.h>

int Max(int x, int y){
    if (x > y) return x;
    else return y;
}

int Min(int x, int y){
    if (x > y) return y;
    else return x;
}

int main(){
    int a, b;
    scanf("%d %d", &a ,&b);
    if (4 <= a && a <= 20 && 4 <= b && b <= 20){
        int m = Max(a, b);
        int k = Min(a, b);
        if (a != b){
            for (int i = (k + 1); i < (m + 2); i++){
                printf("%d\n" , i);
            }
        }
        else printf("%d\n", k + 1);
    }
}

File: Python/dicecup.py
inp = input().split()
if  4 <= int(inp[0]) and int(inp[1]) <= 20:
    b = max(int(inp[0]),int(inp[1]))
    k = min(int(inp[0]),int(inp[1]))
    if int(inp[0]) != int(inp[1]):
        
        for i in range(k + 1 , b + 2):
            print(i)


    else:
        print(k+1)
